Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,702 @@
+id: Endpoint Enrichment - Generic v2.1
+version: -1
+fromversion: 4.1.0
+name: Endpoint Enrichment - Generic v2.1
+description: |-
+  Enrich an endpoint by hostname using one or more integrations.
+  Currently, the following integrations are supported:
+  - Active Directory Query v2
+  - McAfee ePolicy Orchestrator
+  - Carbon Black Enterprise Response v2
+  - Cylance Protect v2
+  - CrowdStrike Falcon Host
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 29bcd70f-1953-4061-84ce-4cde781ad9f7
+    type: start
+    task:
+      id: 29bcd70f-1953-4061-84ce-4cde781ad9f7
+      version: -1
+      name: ""
+      description: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "1":
+    id: "1"
+    taskid: 54895e0d-9904-4e62-8f45-ebd0d17ad5c9
+    type: title
+    task:
+      id: 54895e0d-9904-4e62-8f45-ebd0d17ad5c9
+      version: -1
+      name: Endpoint Products
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+      - "18"
+      - "19"
+      - "20"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 440
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "2":
+    id: "2"
+    taskid: c9799048-288a-40cc-8d2f-c561d0ea2f6b
+    type: condition
+    task:
+      id: c9799048-288a-40cc-8d2f-c561d0ea2f6b
+      version: -1
+      name: Is McAfee ePolicy Orchestrator enabled?
+      description: Checks if there is any active instance of the McAfee ePolicy Orchestrator
```

Comment:
Checks if there is an active instance of the McAfee ePolicy Orchestrator integration enabled.